### PR TITLE
net-misc/httpie: Bump to version 2.0.0

### PR DESCRIPTION
**Pull request by rseichter**
This ebuild also adds support for Python 3.7.

### CODE REVIEW

File: net-misc/httpie/httpie-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/pygments[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.22.0[${PYTHON_USEDEP}]"
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
```
Comment (juippis) [inline net-misc/httpie/httpie-2.0.0.ebuild]: `setuptools` is handled by *distutils-r1* eclass and can be removed here.

File: net-misc/httpie/httpie-2.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		${RDEPEND}
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/pyopenssl[${PYTHON_USEDEP}]
+		dev-python/pytest-httpbin[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+	)"
+
+python_test() {
+	pytest -vv || die "Tests failed with ${EPYTHON}"
+}
```
Comment (juippis) [inline net-misc/httpie/httpie-2.0.0.ebuild]: You should try to use this via the eclass function, `distutils_enable_tests`.

**Comment by rseichter** [comment]
>  DISTUTILS_USE_SETUPTOOLS value is probably incorrect

I added a line to the ebuild, but should the eclass not figure the best value out on its own?

**Comment by juippis** [comment]
> 
> I added a line to the ebuild, but should the eclass not figure the best value out on its own?

Good idea to open a bug about it? :)

I think it's more of a hint to doublecheck, but I have nothing to do with our python project.

**Pull request closed without merging**